packages/web-components/src/components/card-group/card-group.ts

```diff
@@ -185,6 +193,7 @@ class DDSCardGroup extends LitElement {
     const slotClasses = classMap({
       [`${prefix}--card-group--narrow`]: this.gridMode === GRID_MODE.NARROW,
       [`${prefix}--card-group--collapsed`]: this.gridMode === GRID_MODE.COLLAPSED,
+      [`${prefix}--card-group--border`]: this.border,
```

Comment:
Is this separate from the grid mode?

![image](https://user-images.githubusercontent.com/3793636/119404699-c83d0500-bca5-11eb-818f-7d57a96af451.png)

Reply:
Yes, completely independent 👍

Reply:
It's specced out as a grid mode. Not separate or independent of them.

Reply:
Ok, let me fix that 👍

Reply:
Fixed this 👍